IslandLanding/IslandLanding/ViewModel/InfoViewModel.cs

```diff
@@ -71,11 +76,10 @@ private async void CheckusCommandExcute(object obj)
         // An unexpected error occured. No browser may be installed on the device.
       }
     }
-    private async void Play()
+    public static async void Play()
     {
-      await CrossMediaManager.Current.Play("https://ia800806.us.archive.org/15/items/Mp3Playlist_555/AaronNeville-CrazyLove.mp3");
-     // await mediaInfo.Play("https://drive.google.com/file/d/1MSnVohtx-GfvzJJSYycuOq8y22q4o42h/view");
-     // this link not work 
+      var audio =  CrossMediaManager.Current;
+      await audio.PlayFromAssembly("music.mp3", typeof(BaseViewModel).Assembly);
```

Comment:
Please try to use an audio service. Please let me know if it's difficult or not possible

Reply:
your concern here about the nuget or what i dont understand  your point